services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/FlowCrudSteps.java

```diff
@@ -119,6 +122,7 @@
     private TopologyUnderTest topologyUnderTest;
 
     private Set<FlowPayload> flows;
+    private Set<FlowPathPayload> flowPaths = new HashSet<>();
```

Comment:
Set is an **unordered** collection. How do you bind those paths to exact flows at this point? Moreover, if there are two flows with the same paths, it will swallow the path for second flow because Set cannot have duplicates. So you will end up with 2 flows and 1 flow path. What if there is a third flow. 3 flows 2 paths, what next?)